package-lock.json

```diff
@@ -3908,6 +3898,25 @@
         }
       }
     },
+    "glob-parent": {
+      "version": "3.1.0",
```

Comment:
This is changing the version to a lower version that will also cause the audit issues that you're ignoring